src/offset/local/mod.rs

```diff
@@ -16,7 +16,7 @@ use crate::naive::{NaiveDate, NaiveDateTime, NaiveTime};
 use crate::Date;
 use crate::{DateTime, Utc};
 
-#[cfg(unix)]
+#[cfg(all(unix, not(all(target_arch = "wasm32", feature = "wasmbind", target_os = "emscripten"))))]
```

Comment:
Why do we need this? Does the emscripten target identify as `unix`?

Reply:
Yes.

<details>
<summary>Details</summary>

```rs
❯ cat test.rs
#[cfg(unix)]
fn i_am_unix() {}

#[cfg(target_arch = "wasm32")]
fn i_am_wasm32() {}

#[cfg(target_os = "emscripten")]
fn i_am_emscripten() {}

fn main() {}

❯ rustc test.rs --target wasm32-unknown-unknown
warning: function `i_am_wasm32` is never used
 --> test.rs:5:4
  |
5 | fn i_am_wasm32() {}
  |    ^^^^^^^^^^^
  |
  = note: `#[warn(dead_code)]` on by default

warning: 1 warning emitted

❯ rustc test.rs --target wasm32-unknown-emscripten
warning: function `i_am_unix` is never used
 --> test.rs:2:4
  |
2 | fn i_am_unix() {}
  |    ^^^^^^^^^
  |
  = note: `#[warn(dead_code)]` on by default

warning: function `i_am_wasm32` is never used
 --> test.rs:5:4
  |
5 | fn i_am_wasm32() {}
  |    ^^^^^^^^^^^

warning: function `i_am_emscripten` is never used
 --> test.rs:8:4
  |
8 | fn i_am_emscripten() {}
  |    ^^^^^^^^^^^^^^^

warning: 3 warnings emitted

```

</details>